docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+
+  keycloak:
+    image: jboss/keycloak:latest
```

Comment:
DBについての設定しない場合、KeycloakのDB周りってどのような設定になりますか？

Reply:
組み込まれているH2ってやつになるみたいです。
If the DB can't be detected it will default to the embedded H2 database.

You can specify the DB vendor directly with the DB_VENDOR environment variable. Supported values are:

- h2 for the embedded H2 database,

- postgres for the Postgres database,

- mysql for the MySql database.
- mariadb for the MariaDB database.
- oracle for the Oracle database.
- mssql for the Microsoft SQL Server database.